apps/core/src/app.module.ts

```diff
@@ -6,12 +6,14 @@ import { DynamoDbLocalSetupService } from './lifecycle-hooks/dynamodb-local-setu
 import { APP_GUARD } from '@nestjs/core';
 import { CognitoJwtAuthGuard } from './auth/cognito-jwt-auth.guard';
 import { authConfig } from './config/auth.config';
+import { ThrottlerModule } from '@nestjs/throttler';
 
 @Module({
   imports: [
     ConfigModule.forRoot({ load: [databaseConfig], isGlobal: true }),
     ConfigModule.forRoot({ load: [authConfig], isGlobal: true }),
     DashboardsModule,
+    ThrottlerModule.forRoot({ ttl: 10, limit: 100 }),
```

Comment:
We will need to have AWS WAF in front of the Centurion server in production and WAF has support for rate limit as well. But it's good to have another layer of security within the server in case the server is not deployed to the cloud.

Reply:
That's cool!